src/uwtools/tests/test_rocoto.py

```diff
@@ -12,7 +12,7 @@
 from uwtools import rocoto
 from uwtools.config.formats.yaml import YAMLConfig
 from uwtools.exceptions import UWConfigError, UWError
-from uwtools.tests.support import fixture_path
+from uwtools.tests.support import fixture_path, schema_validator
```

Comment:
I appreciate how DRY it is to do the schema checks in this module since we already have config dicts defined here. On the other hand, we have a separate set of Rocoto schema checks in `test_schemas.py` and I think there's some value in having a comprehensive set of schema tests in one place, as it's clear where to find them, to add new ones, etc. I'm torn and am not sure I have a strong opinion. Did you land on placement of the schema checks here after considering both options?

Reply:
I think the comprehensive set of schema checks makes sense in `test_schemas.py`, but I don't think that necessarily needs to be the only place schema checking happens. Here, it ended up being important to ensure that we're validating the small configurations that we're using to check the code, and that validation makes it a much stronger test IMO. 

Rocoto dependencies are the most complicated (I think) schemas and code to check, given that there are so many options and combinations. I think it makes sense to use targeted small subsets of dictionaries to test with like we're doing int `test_rocoto.py`, so I don't see us DRYing things out by just making one big YAML for schema tests and code tests.

Reply:
> by just making one big YAML for schema tests and code tests
Just for the record, I don't think that (one big YAML) is what's done in `test_schemas.py`: There are lots of tiny config fragments that do granular testing of sub-schemas, similar to the dicts in this module. But I have no objects to going this route if you considered both and chose this one intentionally.